src/provider/index.ts

```diff
@@ -33,6 +41,8 @@ export class Main {
     @inject(Inject.TOAST_MANAGER)
     private _toastManager!: ToastManager;
 
+    private interestMap: EventInterestMap = new EventInterestMap;
```

Comment:
done

Reply:
Ah, I meant as in `private readonly _interestMap`, to communicate that the value of `_interestMap` itself doesn't change. Also, we're generally doing `new ClassName()` rather than `new ClassName` elsewhere.